### ISSUE TITLE
Nil safety on getter

### ISSUE THREAD

**Issue by blurey8** [masaushi/accessory#3]
Hello. Can we add nil safety on getter like this protocgen?

https://github.com/simplesteph/protobuf-example-go/blob/master/src/complex/complex.pb.go#L53-L65

```go
func (o *Object) StringField() string {
	if o != nil {
		return o.StringField
	}
	return ""
}

func (o *Object) IntField() int {
	if o != nil {
		return o.IntField
	}
	return 0
}

func (o *Object) AnotherObjectField() *AnotherObject {
	if o != nil {
		return o.AnotherObject
	}
	return nil
}
```

It would be very helpful if this implemented. Thank you!

**Comment by masaushi** [comment]
Thank you for reporting this issue!

Your proposal makes sense.
I'll implement that in some days.

**Comment by masaushi** [comment]
implemented nil safety support at version 0.0.6!
